CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.2.1 (2019-04-25)
+
+- Fixed date range picker component to run validation after user selects date with date picker. [#34](https://github.com/blackbaud/skyux-datetime/pull/34)
```

Comment:
Fixed date range picker component to run validation after users select dates with date picker.

Reply:
Way better, thanks!